terraform/terraform-modules/ecs-ecr/variables.tf

```diff
@@ -0,0 +1,143 @@
+variable "region" {
+  type        = string
+  description = "Region"
+  default     = "us-west-2"
+}
+
+variable "efs_file_system_id" {
+  type        = string
+  description = "EFS File System ID"
+  # default     = "<EFS File System ID>"
+  sensitive   = true
+}
+
+variable "pds_nucleus_ecs_cluster_name" {
+  type        = string
+  description = "The name of the PDS Nucleus ECS cluster"
+  sensitive   = true
+}
+
+variable "pds_data_access_point_id" {
+  type        = string
+  description = "PDS Data Access Point ID"
+  # default     = "<PDS Data Access Point ID>"
+  sensitive   = true
+}
+
+#variable "pds_registry_loader_harvest_ecr_image_path" {
```

Comment:
Can this just be deleted?

If not, could we add a comment explaining why this variable is commented-out?

Reply:
Done